deformable_manipulation_experiment_params/launch/cloth_wafr_params.launch

```diff
@@ -0,0 +1,187 @@
+
+    <arg name="table_leading_edge_x"    value ="-0.185"/>
+
+    <arg name="table_surface_x"         value="0.0"/>
+    <arg name="table_surface_y"         value="0.0"/>
+    <arg name="table_surface_z"         value="$(arg table_height)"/>
+
+    <param name="table_surface_x"       value="$(arg table_surface_x)"  type="double"/>
+    <param name="table_surface_y"       value="$(arg table_surface_y)"  type="double"/>
+    <param name="table_surface_z"       value="$(arg table_surface_z)"  type="double"/>
+
+    <!-- Cloth properties params -->
+
+    <param name="cloth_linear_stiffness"        value="0.8"/>
+
+    <!-- Cloth size params -->
+    
+    <arg name="cloth_x_size"                    value="0.4"/>
+    <arg name="cloth_y_size"                    value="0.4"/>
+    <!-- <arg name="cloth_x_size"                    value="0.5"/> -->
+    <!-- <arg name="cloth_y_size"                    value="0.4"/> -->
+    <!-- <arg name="cloth_num_control_points_x"      value="50"/> -->
+    <!-- <arg name="cloth_num_control_points_y"      value="40"/> -->
+    <arg name="cloth_num_control_points_x"      value="20"/>
+    <arg name="cloth_num_control_points_y"      value="20"/>
+
+    <param name="cloth_x_size"                  value="$(arg cloth_x_size)"                 type="double"/>
+    <param name="cloth_y_size"                  value="$(arg cloth_y_size)"                 type="double"/>
+    <param name="cloth_num_control_points_x"    value="$(arg cloth_num_control_points_x)"   type="int"/>
+    <param name="cloth_num_control_points_y"    value="$(arg cloth_num_control_points_y)"   type="int"/>
+
+    <!-- Obstacle parameters -->
+    
+    <arg name = "cloth_cylinder_height" value="0.3"/>
+    <arg name = "cloth_cylinder_com_x"  value="$(eval table_surface_x - cloth_x_size)"/>
+    <arg name = "cloth_cylinder_com_y"  value="$(arg table_surface_y)"/>
+    <arg name = "cloth_cylinder_com_z"  value="$(eval table_surface_z + (cloth_cylinder_height/2))"/>
+    <arg name = "cloth_cylinder_radius" value="0.1"/>
+    
+    <param name="cloth_cylinder_height" value="$(arg cloth_cylinder_height)"    type="double"/>
+    <param name="cloth_cylinder_com_x"  value="$(arg cloth_cylinder_com_x)"     type="double"/>
+    <param name="cloth_cylinder_com_y"  value="$(arg cloth_cylinder_com_y)"     type="double"/>
+    <param name="cloth_cylinder_com_z"  value="$(arg cloth_cylinder_com_z)"     type="double"/>
+    <param name="wall_com_z"            value="0.32"                            type="double"/> 
+
+    <param name="wall_height"           value="0.3"                             type="double"/>
+    <param name="cloth_cylinder_radius" value="$(arg cloth_cylinder_radius)"    type="double"/>
+
+    <!-- Second wafr cylinder params -->
+
+    <arg name = "wafr_second_cylinder_height"     value="0.8"/>
+    <arg name = "wafr_cylinder_relative_com_x"    value="-0.14"/>
+    <!-- <arg name = "wafr_cylinder_relative_com_x"    value="-0.155"/> -->
```

Comment:
It looks like you didn't go through this file at all to check for changes that should not be pushed to master.